_about/notes/0.7.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Istio 0.7
+
+order: 94
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+For this release, we focused on improving our build and test infrastructures and increasing the
+quality of our tests. As a result, there are no new features for this month.
+
+However, this release does include a large number of bug fixes and performance improvements.
```

Comment:
Should be highlight some of these critical fixes or performance issues?   I think istio helm chart is working for 0.7, which has been broken in prior release. cc @sdake

Reply:
I updated the text.

Note that the main Helm page on istio.io still says "Warning: Helm charts are currently broken in 0.5.0". @sdake is this something you can update?

Reply:
v1alpha3 is there partially too (but maybe we don't want to mention it until 0.8)

there is really 0 changes worth mentioning in the 300+ PRs between 0.6 and 0.7 ?